charts/datafold/charts/server/values.yaml

```diff
@@ -31,7 +31,8 @@ service:
 ingress:
   enabled: true
   className: ""
-  annotations: {}
+  annotations:
+    kubernetes.io/ingress.class: "nginx"
```

Comment:
Redundant after merging #18 

Has the same effect as: https://github.com/datafold/helm-charts/blob/14b0590905a89edef82a09672d4671fefaa9f475/charts/datafold/charts/nginx/templates/ingress.yaml#L9

Reply:
Also, using the annotation is deprecated: https://kubernetes.io/docs/concepts/services-networking/ingress/#deprecated-annotation